rls/src/actions/diagnostics.rs

```diff
@@ -599,10 +599,9 @@ help: consider borrowing here: `&string`"#,
     }
 
     /// ```
-    /// fn main() {
     ///     let mut out = String::new();
     ///     write!(out, "{}", 123);
-    /// }
+
```

Comment:
Do we need this doc-test at all? :astonished:

Reply:
probably not :smile: